src/main/java/com/dev/cinema/model/dto/moviesession/MovieSessionDtoMapper.java

```diff
@@ -0,0 +1,44 @@
+package com.dev.cinema.model.dto.moviesession;
+
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.service.MovieService;
+import com.dev.cinema.service.MovieSessionService;
+import java.time.format.DateTimeFormatter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionDtoMapper {
+    private final MovieSessionService movieSessionService;
```

Comment:
you don't use this service here(you can leave others though)

Reply:
done